index.js

```diff
@@ -92,7 +115,17 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  let newObj = movies.filter((movie)=>{
+    if(genre === movie.title){
+      return movie.title;
+    }else if(genre){
+      return null;
+    }
+  })
+    return newObj;
+}
+
```

Comment:
```suggestion
function filterByGenre(movies, genre) {  
genre = genre.toLowerCase()    // use .toLowerCase to make `genre` case-insensitive.
  let filteredMovies = movies.filter((movie)=>{    //.filter returns a new array, not just one element.
    if(genre.includes(movie.genre.toLowerCase())){     // movie.genre (not movie.title) is where the genres are.
     // you have to use .toLowerCase on movie.genre too 
     //movie.genre has several genres, so use .includes, not "==="
      return true;             
    }else{                              
      return false;
    }
  })
    return filteredMovies;
}

```
**the .filter method works like this:** Each element in the array gets passed into .filter's callback function. Each element returns either `true` or `false` depending on the condition you set. Elements that cause the callback to return `true` make it into the filtered array. Elements that cause the callback to return `false` do NOT make it into the filtered array.